### PR TITLE
TGP-1182: updated commodity code result

### PR DESCRIPTION
**Pull request by vinoocs**
- updated the commodity object to receive multiple descriptions (maximum 3)

- **the functionality for  /update/:recordId/longer-commodity-code-result will be a separate ticket as discussed with Amaka so please ignore that in this**

Page:
image

### CODE REVIEW

File: app/views/HasCorrectGoodsView.scala.html [hunk 1]
```diff
@@ -30,10 +30,18 @@
 
     <h1 class="govuk-heading-l">@messages("hasCorrectGoods.heading", commodity.commodityCode)</h1>
 
-    <p class="govuk-body">@messages("hasCorrectGoods.p1", commodity.description)</p>
+
+    <p class="govuk-body">@messages("hasCorrectGoods.p1")</p>
+
+    <ul class="govuk-body">
+    @commodity.descriptions.zipWithIndex.map { case (description, index) =>
```
Comment (ChrisCookOC) [inline app/views/HasCorrectGoodsView.scala.html]: can do some indenting here to make more readable

File: test/factories/AuditEventFactorySpec.scala [hunk 2]
```diff
@@ -243,7 +244,7 @@ class AuditEventFactorySpec extends SpecBase {
 
           val effectiveFrom = Instant.now
           val effectiveTo   = effectiveFrom.plusSeconds(99)
-          val commodity     = Commodity("030821", "Sea urchins", effectiveFrom, Some(effectiveTo))
+          val commodity     = Commodity("030821", List("Sea urchins"), effectiveFrom, Some(effectiveTo))
```
Comment (ChrisCookOC) [inline test/factories/AuditEventFactorySpec.scala]: If you really wanted to (but its not important) you could have multiple descriptions in the list and then you are testing that the audit event gets the head like you want.

**Review by ChrisCookOC — COMMENTED** [review]
Looks good, minor thoughts added

The Json stuff does make me a bit 🫠, though I know its very hard to make nice looking code when Json is involved here 😄

File: app/views/HasCorrectGoodsView.scala.html [hunk 3]
```diff
@@ -30,10 +30,18 @@
 
     <h1 class="govuk-heading-l">@messages("hasCorrectGoods.heading", commodity.commodityCode)</h1>
 
-    <p class="govuk-body">@messages("hasCorrectGoods.p1", commodity.description)</p>
+
+    <p class="govuk-body">@messages("hasCorrectGoods.p1")</p>
+
+    <ul class="govuk-body">
+        @commodity.descriptions.zipWithIndex.map { case (description, index) =>
+            <li>@messages(description.toLowerCase)</li>
```
Comment (ChrisCookOC) [inline app/views/HasCorrectGoodsView.scala.html]: if this is coming from OTT why does it need to go through messages?

**Review by ChrisCookOC — APPROVED** [review]

**Review by liandra-giacomine — APPROVED** [review]

**Pull request merged**